docs/explanations/modeling_extensions/surrogate/api/alamopy/alamopy-options.rst

```diff
@@ -0,0 +1,196 @@
+ALAMOPY.ALAMO Options
+=====================
+
+This page lists in more detail the ALAMOPY options and the relation of ALAMO and ALAMOPY.
+
+.. contents::
+    :depth: 3
+
```

Comment:
The first thing we need here is instructions on how to install ALAMO, emphasizing that it is an external package that needs to be installed manually. We should also mention the need for a license. Most of this can be done with a link to the ALAMO docs. See https://github.com/IDAES/idaes-pse/issues/85.

We should also mention that more detail on ALAMO options can be found in the ALAMO docs.